content/intro-to-writing-data.md

```diff
@@ -259,23 +265,24 @@ const signature = await web3.sendAndConfirmTransaction(
 console.log(`✅ Transaction completed! Signature is ${signature}`)
 ```
 
-### 5. Airdrop
+### 4. Run the program
+Now call the `pingProgram()` 
 
-Now run the code with `npx esrun send-ping-instruction.ts` and see if it works. You may end up with the following error in the console:
-
-```
-> Transaction simulation failed: Attempt to debit an account but found no record of a prior credit.
+```typescript
+async function main() {
+    const payer = getKeypairFromEnvironment("SECRET_KEY");
+    console.log(` ✅ Loaded payer keypair ${payer.publicKey.toBase58()}`);
+    await pingProgram(connection, payer);
+}
 ```
 
-If you get this error, it’s because your keypair is brand new and doesn’t have any SOL to cover the transaction fees. Let’s fix this by adding the following line before the call to `sendPingTransaction()`:
-
 ```typescript
-await connection.requestAirdrop(payer.publicKey, web3.LAMPORTS_PER_SOL*1)
+main().catch((err) => {
```

Comment:
Remember `esrun` supports `await` at the top level. 

So we can do:

```
try {
  await doThings();
} catch (error) {
   @console.error(`❌ Something went wrong ${error.message}`)
}
```

See `CONTRIBUTING.md` for details!